### PR TITLE
fix: support both types of "repository" in package.json

### PR DESCRIPTION
**Pull request by capaj**
add support for package.json which has "repository" as string, not an object

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: lib/commands/getRepositoryUrl.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module.exports = function (git) {
```
Comment (climba03003) [inline lib/commands/getRepositoryUrl.js:1]: `commands` folder should only contain commands

**Review by climba03003 — CHANGES_REQUESTED** [review]

File: lib/commands/getRepositoryUrl.js (hunk 1 above)
Comment (capaj) [reply lib/commands/getRepositoryUrl.js]: should be done now

**Review by Eomm — APPROVED** [review]

**Comment by Eomm** [comment]
The CI is reporting a regression on coverage, I think the error test on `getRepositoryUrl` is missing

> ERROR: Coverage for lines (99.69%) does not meet global threshold (100%)
ERROR: Coverage for branches (94.93%) does not meet global threshold (95%)
ERROR: Coverage for statements (99.7%) does not meet global threshold (100%)

**Comment by capaj** [comment]
Yeah I will add it.

**Review by Eomm — APPROVED** [review]

**Review by climba03003 — APPROVED** [review]
LGTM.

**Pull request merged**